.github/workflows/collector.yml

```diff
@@ -39,6 +44,7 @@ jobs:
 
     env:
       PLATFORM: linux/${{ matrix.arch }}
+      CONTEXT_DRIVERS_DIR: ${{ github.workspace }}/kernel-modules/container/kernel-modules
```

Comment:
Think if we set this to download the drivers to somewhere like `${{ github.workspace }}/collector/kernel-modules` and we put the `LICENSE-kernel-modules.txt ` file in that directory as well renamed to `LICENSE`, we can change [this line in the Dockerfile](https://github.com/stackrox/collector/blob/6be77738ebe0c5f5994597fd5f630215d08505b3/collector/container/Dockerfile#L29) to directly copy the license and drivers, WDYT? (We might also need to change the Konflux dockerfile)